packages/lab/src/utils/useIsViewportLargerThanBreakpoint.ts

```diff
@@ -7,10 +7,10 @@ import {
 export const useIsViewportLargerThanBreakpoint = (
   targetedBreakpoint: keyof Breakpoints
 ): boolean => {
+
   const orderedBreakpoints = useOrderedBreakpoints();
   const index = orderedBreakpoints.indexOf(targetedBreakpoint);
-  const allPreviousBreakpoints = orderedBreakpoints.slice(0, index + 1);
-
   const currentBreakpoint = useCurrentBreakpoint();
-  return allPreviousBreakpoints.includes(currentBreakpoint);
+  const currentBreakpointIndex = orderedBreakpoints.indexOf(currentBreakpoint);
+  return index >= currentBreakpointIndex
```

Comment:
I thought doing this comparison of seeing if the index of the current breakpoint was less than or equal to target breakpoint was more simplistic - thoughts on if this prevents any use cases ?